src/bundles.ts

```diff
@@ -209,12 +209,14 @@ export async function getSupportedFiles(
   const supportedFilesFromApi = resp.value;
   //Given supported languages from 'registy'
   if (languages) {
-    const supportedFiles: SupportedFiles = {} as any;
+    const supportedFiles = {} as SupportedFiles;
     supportedFiles.configFiles = supportedFilesFromApi.configFiles;
     supportedFiles.extensions = languages;
+
     //For verification only
+    const userSupportedExtensions = supportedFilesFromApi.extensions.map(e => e.toLowerCase());
```

Comment:
enh: A comment explaining the reasoning here might be helpful for future maintainers.